videozen-frontend/src/components/AllPlaylists/CreatedPlaylists/CreatedPlaylists.jsx

```diff
@@ -0,0 +1,74 @@
+import { Link } from "react-router-dom";
+
+import { useData } from "../../../context/data-context";
```

Comment:
add `index.js` in context folder to make imports small
```suggestion
import { useData } from "../../../context";
```